src/filterObj.js

```diff
@@ -0,0 +1,29 @@
+var _curry2 = require('./internal/_curry2');
+var _filterObj = require('./internal/_filterObj');
+
+/**
+ * Returns a new object containing only those key-value pairs of which the
+ * values match a given predicate function.
+ * The predicate function is passed one argument: *(value)*.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig (v -> Boolean) -> {k, v} -> {k, v}
+ * @param {Function} fn The function called per key.
+ * @param {Object} obj The object to iterate over the keys.
+ * @return {Object} The new filtered object.
+ * @example
+ *
+ *      var isPositive = function(n) {
+ *        return n > 0;
+ *      };
+ *      R.filterObj(isPositive, {a: 1, b: 2, c: -1, d: 0, e: 5}); //=> {a: 1, b: 2, e: 5}
+ *
+ *      var containsBackground = function(x) {
+ *        return x.bgcolor;
+ *      };
+ *      var colors = {1: {color: 'read'}, 2: {color: 'black', bgcolor: 'yellow'}};
+ *      R.filterObj(containsBackground, colors); //=> {2: {color: 'black', bgcolor: 'yellow'}}
+ */
+module.exports = _curry2(_filterObj);
```

Comment:
unless `_filterObj` is being reused somewhere else, might as well inline the definition here